packages/wonder-blocks-icon/src/components/phosphor-icon.tsx

```diff
@@ -91,12 +99,16 @@ export const PhosphorIcon = React.forwardRef(function PhosphorIcon(
         testId,
         className,
         role,
+        sizingUnit = "px",
         ...sharedProps
     } = props;
 
-    const pixelSize = viewportPixelsForSize(size);
+    const iconSize =
```

Comment:
On another note, there is also the `Icon` component in the icon package for custom icons. It's using `sizing` tokens so it is using rems already so that's handled already!